### DISCUSSION TITLE
Access to TranslocoService within custom TranslocoMissingHandler

### DISCUSSION THREAD

**Discussion by dasMoeppi** [ngneat/transloco#810] [Q&A] [answered]
Hey,
the requirement for having access to the `TranslocoService` within a custom `TranslocoMissingHandler` has been addressed before, see https://github.com/jsverse/transloco/issues/555 and https://github.com/jsverse/transloco/issues/645.

There has already been posted a workaround on how to deal with it (injecting the service dynamically). Unfortunately, the solution/working example is not available anymore. Can anybody provide me with a solution maybe?

Kind regards

**Comment by shaharkazaz** [comment, marked as answer]
I've updated the link in #555, you can find a usage example [here](https://codesandbox.io/p/sandbox/using-translocoservice-inside-missing-handler-555-645-tvl4dp?file=%2Fsrc%2Fapp%2Fmissing-handler.ts%3A3%2C27).

**Reply by dasMoeppi** [reply]
Awesome, thank you very much for your fast support!
